integration-libs/cdc/schematics/add-cdc/schema.ts

```diff
@@ -0,0 +1,7 @@
+import { LibraryOptions } from '@spartacus/schematics';
+
+export interface Schema extends LibraryOptions {
+  baseSite?: string;
```

Comment:
if my assumption is correct https://github.com/SAP/spartacus/pull/15756/files#r875015142, then remove the optional param and make it required